src/server/établissement-accompagnement/infra/apiÉtablissementPublic.repository.test.ts

```diff
@@ -32,44 +41,30 @@ describe('ApiÉtablissementPublicRepository', () => {
 				expect(result).toEqual(expected);
 			});
 		});
-
-		describe('lorsque l‘api retourne une erreur 404', () => {
-			it('renvoie une erreur demande incorrecte', async () => {
+		describe('lorsqu‘il y a une erreur lors de la recherche des établissements public', () => {
+			it('log les informations de l’erreur et retourne une erreur métier associée', async () => {
```

Comment:
ca aurait été de dire dans le it que c'est une "demande incorrecte"

Reply:
on s'est mis d'accord pour utiliser cette phrase là suite à un retour sur une autre PR